tools/validateJSON.js

```diff
@@ -80,6 +80,7 @@ function validate(json, schema, schemaFolderPath) {
     else {
         tv4.addSchema(json);
         tv4.addSchema(schema);
+       console.log("%s %s %s", json, schema, schemaFolderPath);
```

Comment:
>console.log("%s %s %s", json, schema, schemaFolderPath); [](start = 7, length = 56)

looks like this is causing malformed output: (see the [object Object].

C:\code\azure-resource-manager-schemas\schemas\2014-04-01\Microsoft.BizTalkServices.json
[object Object] [object Object] C:\code\azure-resource-manager-schemas\schemas
    Using schema: C:\code\azure-resource-manager-schemas\tools\ResourceMetaSchema.json
        Passed
[object Object] [object Object] C:\code\azure-resource-manager-schemas\schemas
    Using schema: [http://json-schema.org/draft-04/schema](http://json-schema.org/draft-04/schema)
        Passed